### ISSUE TITLE
Unable to start activity ComponentInfo in MainActivity

### ISSUE THREAD

**Issue by lamiajoyee** [rjsvieira/floatingMenu#1]
Just imported the gradle and placed the xml in MainActivity. The project is being built properly but crashes right on start
**Project build.gradle**

    buildscript {
            repositories {
                jcenter()
            }
            dependencies {
                classpath 'com.android.tools.build:gradle:2.2.3'
            }
        }

    allprojects {
        repositories {
            jcenter()
            maven { url 'https://jitpack.io' }
        }
    }

    task clean(type: Delete) {
        delete rootProject.buildDir
    }

**Module build.gradle**
```
apply plugin: 'com.android.application'

android {
    compileSdkVersion 25
    buildToolsVersion "25.0.2"
    defaultConfig {
        applicationId "com.example.lamia.faq"
        minSdkVersion 15
        targetSdkVersion 25
        versionCode 1
        versionName "1.0"
        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
    }
    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
        }
    }
    dexOptions {
        incremental false
    }
}

dependencies {
    compile fileTree(include: ['*.jar'], dir: 'libs')
    androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
        exclude group: 'com.android.support', module: 'support-annotations'
    })
    compile 'com.android.support:appcompat-v7:25.1.1'
    compile 'com.github.rjsvieira:floatingMenu:1.0.0'
    testCompile 'junit:junit:4.12'

}

```
**activity_main.xml**
```
<?xml version="1.0" encoding="utf-8"?>
<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:tools="http://schemas.android.com/tools"
    android:id="@+id/activity_main"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    android:paddingBottom="@dimen/activity_vertical_margin"
    android:paddingLeft="@dimen/activity_horizontal_margin"
    android:paddingRight="@dimen/activity_horizontal_margin"
    android:paddingTop="@dimen/activity_vertical_margin"
    tools:context="com.example.lamia.faq.MainActivity">

    <TextView
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:text="Hello World!" />


    <rjsv.FloatingMenuButton.FloatingMenuButton.FloatingMenuButton
        android:id="@+id/my_floating_button"
        android:layout_width="40dp"
        android:layout_height="40dp"
        android:scaleType="centerInside"
        >

        <rjsv.FloatingMenuButton.FloatingMenuButton.subbutton.FloatingSubButton
            android:id="@+id/sub_button_1"
            android:layout_width="50dp"
            android:layout_height="50dp"
            />

        <rjsv.FloatingMenuButton.FloatingMenuButton.subbutton.FloatingSubButton
            android:id="@+id/sub_button_2"
            android:layout_width="50dp"
            android:layout_height="50dp"
            />

        <rjsv.FloatingMenuButton.FloatingMenuButton.subbutton.FloatingSubButton
            android:id="@+id/sub_button_3"
            android:layout_width="50dp"
            android:layout_height="50dp"
            />

        <rjsv.FloatingMenuButton.FloatingMenuButton.subbutton.FloatingSubButton
            android:id="@+id/sub_button_4"
            android:layout_width="50dp"
            android:layout_height="50dp"
            />

        <rjsv.FloatingMenuButton.FloatingMenuButton.subbutton.FloatingSubButton
            android:id="@+id/sub_button_5"
            android:layout_width="50dp"
            android:layout_height="50dp"
            />

    </rjsv.FloatingMenuButton.FloatingMenuButton.FloatingMenuButton>
</RelativeLayout>

```
android.view.InflateException: Binary XML file line #19: Error inflating class rjsv.FloatingMenuButton.FloatingMenuButton.FloatingMenuButton

 Caused by: java.lang.ClassNotFoundException: Didn't find class "rjsv.FloatingMenuButton.FloatingMenuButton.FloatingMenuButton" on path: DexPathList[[zip file "/data/app/com.example.lamia.faq-1/base.apk"],nativeLibraryDirectories=[/vendor/lib, /system/lib]]

**Comment by rjsvieira** [comment]
Hello lamiajoyee,
Thank your for submitting this error.  The provided xml snippet was in fact incorrect. 
I have fixed this and you can now find the correct .xml implementation on the READ.me file (or below if you need)

Regards,

```xml
<rjsv.floatingmenu.floatingmenubutton.FloatingMenuButton
  android:id="@+id/my_floating_button"
  android:layout_width="40dp"
  android:layout_height="40dp"
  android:scaleType="centerInside"
  circleview:layout_constraintBottom_toBottomOf="parent"
  circleview:layout_constraintLeft_toLeftOf="parent"
  circleview:layout_constraintRight_toRightOf="parent"
  circleview:layout_constraintTop_toTopOf="parent"
  FloatingMenuButtonActionButton:animationType="radial"
  FloatingMenuButtonActionButton:dispositionEndAngle="360"
  FloatingMenuButtonActionButton:dispositionStartAngle="0"
  FloatingMenuButtonActionButton:subActionButtonRadius="100">

  <rjsv.floatingmenu.floatingmenubutton.subbutton.FloatingSubButton
    android:id="@+id/sub_button_1"
    android:layout_width="50dp"
    android:layout_height="50dp"
    android:background="@drawable/one"/>

  <rjsv.floatingmenu.floatingmenubutton.subbutton.FloatingSubButton
    android:id="@+id/sub_button_2"
    android:layout_width="50dp"
    android:layout_height="50dp"
    android:background="@drawable/two"/>

  <rjsv.floatingmenu.floatingmenubutton.subbutton.FloatingSubButton
    android:id="@+id/sub_button_3"
    android:layout_width="50dp"
    android:layout_height="50dp"
    android:background="@drawable/three"/>

  <rjsv.floatingmenu.floatingmenubutton.subbutton.FloatingSubButton
    android:id="@+id/sub_button_4"
    android:layout_width="50dp"
    android:layout_height="50dp"
    android:background="@drawable/four"/>

  <rjsv.floatingmenu.floatingmenubutton.subbutton.FloatingSubButton
    android:id="@+id/sub_button_5"
    android:layout_width="50dp"
    android:layout_height="50dp"
    android:background="@drawable/five"/>

</rjsv.FloatingMenuButton.FloatingMenuButton.FloatingMenuButton>
```

**Comment by lamiajoyee** [comment]
Thank you for the prompt fix in the readme

**Comment by rjsvieira** [comment]
Fixed in : 13e316eb941775f51cd563160e9c1977c2518ed8
